Assets/Scripts/World/PickupPup/DogAI/DogAI.cs

```diff
@@ -119,7 +119,7 @@ IEnumerator wander()
             if (target == GetComponent<RectTransform>().anchoredPosition)
             {
                 float cTheta = UnityEngine.Random.Range(0, 2 * Mathf.PI);
-                float cRadius = Screen.width * Mathf.Sqrt(UnityEngine.Random.Range(0f, 1f));
+                float cRadius = Screen.width / 2 * Mathf.Sqrt(UnityEngine.Random.Range(0f, 1f));
```

Comment:
Yeah, just the `/ 2` as a scalar variable if we were to implement a tuning var there.